pyspark/bigdl/contrib/onnx2bigdl/load_model.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import onnx
+from .load_graph import OnnxGraph
+
+
+class OnnxModel(object):
+	def __init__(self, file_path=None):
+		if file_path:
+			self._model_proto = onnx.load_model(file_path)
+		else:
+			self._model_proto = None
+
+	def get_model_proto(self):
+		return self._model_proto
+
+	def load_model(self, model_proto):
```

Comment:
why do we need to create a separate method since per below you just call `bgraph.load_graph(graph_proto)` ?   there is no extra logic inside this method per my understanding

Reply:
There are other model meta data besides graph.
Though we just load the graph for now, I think its necessary to load all of them into the model, in case users want to check the loaded onnx model.